js/test/tests.js

```diff
@@ -307,3 +307,10 @@ test("twttr.txt.extractUrls", function() {
   equal(twttr.txt.extractUrls(message_with_hyphenated_url)[0], "hyphenated-url.com", "Should extract full url with hyphen.");
   equal(twttr.txt.extractUrls(message_with_www_hyphenated_url)[0], "www.123-hyphenated-url.com", "Should extract full url with hyphen.");
 });
+
+test("twttr.txt.getTweetLength", function() {
+  equal(twttr.txt.getTweetLength(""), 0, "empty should be zero length.");
+  equal(twttr.txt.getTweetLength("sample tweet"), 12, "Should extract full url with hyphen.");
```

Comment:
The test's message 'full url with hyphen' seems to not match the tweet content 'sample tweet'. Once this is fixed I'll merge; extra tests that clarify behavior are always appreciated, thanks!